### ISSUE TITLE
Set a readable build name

### ISSUE THREAD

**Issue by martinandersson** [martinanderssondotcom/box-ubuntu-budgie-18-x64#2]
See https://www.packer.io/docs/templates/builders.html#named-builds.

Currently, the VM image once imported to VirtualBox will be named something weird, like "default_123123123_123123123".

**Comment by martinandersson** [comment]
Will not fix, for a number of reasons.

Reason 1) Setting a build name, as it turned out after a test run, will change the command used to build the box. With a build name specified, the name has to be used instead of the builder type. So the command would go from:

    packer build -only virtualbox-iso template.json

to (assuming the name "vbox-ubuntu-18")..

    packer build -only vbox-ubuntu-18 template.json

"virtualbox-iso" is the builder type which is officially documented. Also, inferring the alternative type "vmware-iso" is a no-brainer.

However, using specific names must be documented and is no longer an official type name; i.e., it complicates how to build the project.

One can not use the same build name for both or Packer will crash, nor is it possible - after having specified a build name - to use the build type on the command line (hey, thanx HashiCorp).

2\) Using a build name had \*drum roll* little effect to no effect. During build-time, the only thing that changed was the machine name "default". Packer will still add a bunch of random numbers. But then, the build name is lost. When the box is imported to the provider, the box will yet again be called "default" together with a bunch of random numbers.

So bottom line is this: Setting a build name did not improve shit. In fact, it only complicated things.
